src/transactions/TransactionTable.js

```diff
@@ -40,6 +62,13 @@ function TransactionTable({ isCreating = false, onStopCreating }) {
 }
 
 TransactionTable.propTypes = {
+  filters: PropTypes.exact({
+    categoryId: PropTypes.string,
+    payee: PropTypes.string,
+    startDate: PropTypes.number,
+    endDate: PropTypes.number,
+  }).isRequired,
+  // date: new Date(formData.date)
```

Comment:
Nice job with the prop types! Rather than making the filters prop required, lets give it a default value of an empty object. You can see an example of setting a default value with the isCreating prop.

We also need a JSDoc comment explaining what this prop is and how to use it. Think about what you'd want to know if you needed to use this component and couldn't see the code.